abb_irb120_support/package.xml

```diff
@@ -0,0 +1,41 @@
+<package>
+  <name>abb_irb120_support</name>
+  <version>0.2.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the ABB IRB 120
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for ABB IRB 120 manipulator. This currently includes the base model.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in
+      the ABB data sheets.  All URDFs / XACROs are based on the
```

Comment:
Which data sheet in particular did you use? Ideally we'd provide a link (or document nr), so users can look it up and check their own robot.